src/scene.rs

```diff
@@ -127,20 +228,40 @@ impl<'a> Node<'a> {
 
     /// Returns the node's unit quaternion rotation in the order `[x, y, z, w]`,
     /// where `w` is the scalar.
+    #[deprecated(since = "0.9.1", note = "Use `transform().decomposed()` instead.")]
     pub fn rotation(&self) -> [f32; 4] {
         self.json.rotation.0
     }
 
     /// Returns the node's non-uniform scale.
+    #[deprecated(since = "0.9.1", note = "Use `transform().decomposed()` instead.")]
     pub fn scale(&self) -> [f32; 3] {
         self.json.scale
     }
 
     /// Returns the node's translation.
+    #[deprecated(since = "0.9.1", note = "Use `transform().decomposed()` instead.")]
     pub fn translation(&self) -> [f32; 3] {
         self.json.translation
     }
 
+    /// Returns the node's transform.
+    pub fn transform(&self) -> Transform {
+        if let Some(matrix) = self.json.matrix.clone() {
+            unsafe {
+                Transform::Matrix {
+                    matrix: mem::transmute(matrix),
```

Comment:
This loos like it does two extra copies compared to the deprecated `matrix` method (clone + transmute) - is that right?

Reply:
I don't think so.

`mem::transmute` moves bytes and `matrix` is `Copy` anyway.

Reply:
Oops... I pressed review button by accident. Not sure what that does.